.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,8 +30,17 @@
 1. ...
 2. ...
 
+### Release notes
+
+For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud ECE-tools release notes](https://devdocs.magento.com/cloud/release-notes/ece-release-notes.html).
+
+### Associated documentation updates
+<!--
+ If your proposed update requires user documentation, submit a PR to the Magento DevDocs repository. For extensive updates requiring assistance, submit an issue to DevDocs. See https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md.
+ -->
+Add link to Magento DevDocs PR or Issue, if needed.
+
 ### Contribution checklist
  - [ ] Pull request has a meaningful description of its purpose
```

Comment:
Please can we have this clarified that it is not sufficient to just put a link to a JIRA ticket as the description? This is often done and the issue is that Community Maintainers like myself do not have access to these tickets.

Reply:
I added clarification in the Description comments.